<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find nav)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find nav)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find nav)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav)/config/dwa_local_planner_params.yaml" command="load" />
    <rosparam file="$(find nav)/config/move_base_params.yaml" command="load" />
    <rosparam file="$(find nav)/config/global_planner_params.yaml" command="load" />
    <remap from="cmd_vel" to="nav_cmd_vel"/>
  </node>
</launch>
